### DISCUSSION TITLE
Debugging a single Go file without launch.json

### DISCUSSION THREAD

**Discussion by xmha97** [golang/vscode-go#1245] [Help]
Hi, I can debugging a single Python (py) file without launch.json
Can I debugging a single Go file without launch.json?
![image](https://user-images.githubusercontent.com/23485114/108697831-9a1a1e80-7518-11eb-85f9-4a039f3e5dd0.png)

**Comment by hyangah** [comment]
@xmha97 yes, the extension provides the default debug launch configuration and users can debug their programs and tests without launch.json. From the screenshot, I guess the program couldn't compile because from go1.16, go commands run in module mode. You need to run `go mod init <your module name>` from the workspace root and `go.mod` file. 
See https://golang.org/doc/code#Command for the details.

ps. Sorry for this late response. For some reason, I missed this post. :-(
